backends/xnnpack/third-party/generate-xnnpack-wrappers.py

```diff
@@ -8,16 +8,19 @@
 
 BANNER = "Auto-generated by generate-wrappers.py script. Do not modify"
 WRAPPER_SRC_NAMES = {
```

Comment:
@digantdesai I retook inventory of all the prod microkernel sources and sources that we were/weren't using and cleaned this up. Also added new ones that the new update introduces. Do you mind taking a look here